tw-NEDa-frontend/src/app/components/home/home.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import {async, ComponentFixture, TestBed} from "@angular/core/testing";
```

Comment:
This is a test class that unless you plan to actually write some tests is useless.